chall01/ismelich.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import sys
+
+morse_code = {
+    'A':'.-',
+    'B':'-...',
+    'C':'-.-.',
+    'D':'-..',
+    'E':'.',
+    'F':'..-.',
+    'G':'--.',
+    'H':'....',
+    'I':'..',
+    'J':'.---',
+    'K':'-.-',
+    'L':'.-..',
+    'M':'--',
+    'N':'-.',
+    'O':'---',
+    'P':'.--.',
+    'Q':'--.-',
+    'R':'.-.',
+    'S':'...',
+    'T':'-',
+    'U':'..-',
+    'V':'...-',
+    'W':'.--',
+    'X':'-..-',
+    'Y':'-.--',
+    'Z':'--..',
+}
+
+
+if len(sys.argv) != 2:
+    print ('usage: ./ismelich.py <a-zA-Z string>')
+    sys.exit(1)
+
+str = sys.argv[1]
+for x in str:
+    av = ord(x)
+    if av < 32 or 32 < av < 65 or 90 < av < 97 or av > 122:
```

Comment:
you can use isalpha() and replace() to avoid looping on every char of the string and using ascii value to check
```python
if not sys.argv[1].replace(" ", "").isalpha()
```